src/quicktype-core/language/Python.ts

```diff
@@ -408,6 +410,20 @@ export class PythonRenderer extends ConvenienceRenderer {
         return this.typeHint(" -> ", this.withTyping(type));
     }
 
+    protected sortClassProperties(properties: ReadonlyMap<string, ClassProperty>): ReadonlyMap<string, ClassProperty> {
+        if (this.getAlphabetizeProperties()) {
+            return mapSortBy(properties, (_p: ClassProperty, jsonName: string) => {
+                return jsonName;
+            });
```

Comment:
Sorry I'm being super pedantic here, but this method should only handle the new cast.  In this case, and the third (`return properties`) it should just do `return super.sortClassProperties(properties)`